types/ldapjs/ldapjs-tests.ts

```diff
@@ -12,6 +12,13 @@ let clientWithMultipleURLs = ldap.createClient({
     ]
 });
 
+// @ts-expect-error
+client.port === 1389;
+client.port === '1389';
+client.host === '127.0.0.1';
+// $ExpectType boolean
+clientWithMultipleURLs.secure;
```

Comment:
Suggestion. Just for the sake of correctness:
```
client.on('connect', socket => {
    // @ts-expect-error
    client.port === 1389;
    client.port === '1389';
    client.host === '127.0.0.1';
});
clientWithMultipleURLs.on('connect', socket => {
    // $ExpectType boolean
    clientWithMultipleURLs.secure;
});
```

Reply:
excellent suggestion!